app/src/main/java/com/android/unio/ui/authentication/EmailVerification.kt

```diff
@@ -48,7 +49,11 @@ import com.google.firebase.auth.auth
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun EmailVerificationScreen(navigationAction: NavigationAction, userViewModel: UserViewModel) {
+fun EmailVerificationScreen(
+    navigationAction: NavigationAction,
+    authViewModel: AuthViewModel,
+    onEmailVerified: () -> Unit
```

Comment:
Modify the javadoc to reflect the signature change

Reply:
Good catch ! You are right we often forget to update the java doc and forget what we have done.